lib/shipwire/products.rb

```diff
@@ -32,16 +32,16 @@ def params_runner(params)
       abridge(params)
     end
 
-    def payload_runner(payload)
-      [payload].flatten.each_with_object([]) do |item, pl|
+    def body_runner(body)
+      [body].flatten.each_with_object([]) do |item, pl|
         pl << abridge(item)
       end
     end
 
-    def abridge(payload)
+    def abridge(body)
       classification = product_classification || {}
 
-      data          = RecursiveOpenStruct.new(payload).to_h
+      data          = RecursiveOpenStruct.new(body).to_h
       data_override = RecursiveOpenStruct.new(classification).to_h
```

Comment:
That's a good question. I suppose that if we remove it from Response we can remove it from here, too.